src/Modules/UI/Dnn.EditBar.UI/Controllers/ContentEditorManager.cs

```diff
@@ -161,40 +159,11 @@ protected override void CreateChildControls()
                 {
                     continue;
                 }
-

```

Comment:
@zyhfish  please review this change

Reply:
These code can't be removed, there have issues related to these changes.

Reply:
Hi @zyhfish, those changes seems to be done for https://dnntracker.atlassian.net/browse/CONTENT-5817, which is also running fine after this fix on framework 4.5